src/component/view/bootstrap/entity-form.html

```diff
@@ -0,0 +1,19 @@
+<template>
+  <aurelia-form behavior.bind="behavior">
+    <div>
```

Comment:
Why this div? I don't think we need it (not according to the bootstrap docs anyway)